bumble/crypto.py

```diff
@@ -25,19 +25,15 @@
 
 import logging
 import operator
-
 import secrets
-from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
-from cryptography.hazmat.primitives.asymmetric.ec import (
-    generate_private_key,
-    ECDH,
-    EllipticCurvePrivateKey,
-    EllipticCurvePublicNumbers,
-    EllipticCurvePrivateNumbers,
-    SECP256R1,
-)
-from cryptography.hazmat.primitives import cmac
-from typing import Tuple
+
+try:
```

Comment:
This could be merged in `bumble/crypto/__init__.py`, and the other files in `bumble/crypto` instead of `bumble/_crypto`.

Reply:
Do you mean we don't keep `bumble/crypto.py` anymore, and move original `bumble/crypto.py` to `bumble/crypto/__init__.py` and import the backends there?

Reply:
Yes. I think we can also remove the _ prefix for the files in the crypto directory, they could be simply named builtin.py and cryptography.py

Reply:
Done. My initial idea is to hide some implementation details, but it should still be fine to expose them.